grobian@gentoo.org:
info->license is NULL for e.g. the WMF module, causing a crash on
platforms which' libc doesn't deal with null arguments to the sprintf
family, like Solaris.

--- ./gdk-pixbuf/queryloaders.c
+++ ./gdk-pixbuf/queryloaders.c
@@ -129,7 +129,7 @@
 	g_printf("\"%s\"\n", path);
 	g_printf ("\"%s\" %u \"%s\" \"%s\" \"%s\"\n", 
 		  info->name, info->flags, 
-		  info->domain ? info->domain : GETTEXT_PACKAGE, info->description, info->license);
+		  info->domain ? info->domain : GETTEXT_PACKAGE, info->description, info->license ? info->license : "(unknown)");
 	for (mime = info->mime_types; *mime; mime++) {
 		g_printf ("\"%s\" ", *mime);
 	}
